src/lib/plugin_apis/part.api

```diff
@@ -27,6 +27,20 @@ typedef enum {
     BD_PART_DISK_FLAG_GPT_PMBR_BOOT = 1
 } BDPartDiskFlag;
```

Comment:
...some *of* these "flags"... (in the commit message)

Reply:
Fixed